tools/mo/openvino/tools/mo/utils/ir_reader/layer_to_class.py

```diff
@@ -405,6 +405,9 @@ def copy_graph_with_ops(graph: Graph) -> Graph:
                 'Const node {} not properly corrected to appropriate data node'.format(op.soft_get('name'))
             op.out_node(0)['correct_data_type'] = True
 
+            if op.has_and_set('rt_info'):
+                op.out_node(0)['save_rt_info'] = op.rt_info
```

Comment:
Can we use 'rt_info' insted of 'save_rt_info'? And not to create extra attribute.

Reply:
Yes, we can. I made this attribute to differ 'save_rt_info' of data node from 'rt_info' of const node but it is not strongly necessary.

Reply:
Updated.